mu/virtual_environment.py

```diff
@@ -782,17 +804,21 @@ def create_venv(self):
         logger.info("Virtualenv name: {}".format(self.name))
 
         env = dict(os.environ)
-        ok, output = self.run_subprocess(
-            sys.executable,
-            "-I",
-            "-m",
-            "virtualenv",
-            "-p",
-            sys.executable,
-            "-q",
-            self.path,
-            env=env,
+        args = filter(
+            None,
+            (
+                sys.executable,
+                "-I",
+                "-m",
+                "virtualenv",
+                "-p",
+                sys.executable,
+                "-q",
+                "" if self._is_windows else "--symlinks",

```

Comment:
Turns out the `--symlinks` flag was only added to the `virtualenv` pacakge in version 20.0.0.
We currently support v16 and Debian/Raspbian Buster ships with v15, so in those environments this fails the virtualenv creation.

Reply:
https://github.com/mu-editor/mu/issues/2256